### PR TITLE
Contig2fastg Update

### PR DESCRIPTION
**Pull request by jraysajulga**
Hello. I have updated the descriptions for contig2fastg. Hoping to get this on Galaxy.eu.

### CODE REVIEW

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml [hunk 1]
```diff
@@ -14,11 +14,11 @@
         '${contigs}' > '${fastg}'
     ]]></command>
     <inputs>
-        <param name="contigs" type="data" format="fastq,fastqsanger,fasta,fastq.gz,fastqsanger.gz" multiple="true" label="Single-end file(s)" help="FASTQ/FASTA/FASTQ.GZ files accepted" />
-        <param name="kmer" type="integer" value="99" label="K-mer length" />
+        <param name="contigs" type="data" format="fasta" label="Contig file" help="Ex: k99.contigs.fa" />
```
Comment (bgruening) [inline tools/megahit_contig2fastg/megahit_contig2fastg.xml]: This example does not make sense in the Galaxy context as the user never sees a filename.

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml [hunk 2]
```diff
@@ -14,11 +14,11 @@
...
+        <param name="kmer" type="integer" value="99" label="K-mer length" help="Usually indicated within the filename (when derived from MEGAHIT)"/>
```
Comment (bgruening) [inline tools/megahit_contig2fastg/megahit_contig2fastg.xml]: I'm not sure this help is useful either as the filename is not seen by the user.

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<?xml version='1.0' encoding='utf-8'?>
-<tool id="megahit_contig2fastg" name="MEGAHIT Contig2fastg" version="@VERSION@">
-    <description>for converting MEGAHIT's contigs to fastg</description>
+<?xml version='0.9' encoding='utf-8'?>
+<tool id="megahit_contig2fastg" name="megahit contig2fastg" version="@VERSION@">
+    <description>for converting MEGAHIT's contigs (.fa) to assembly graphs (.fastg)</description>
```
Comment (bgruening) [inline tools/megahit_contig2fastg/megahit_contig2fastg.xml]: what about `for converting MEGAHIT's contigs to assembly graphs`?

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml [hunk 4]
```diff
@@ -1,8 +1,8 @@
-<?xml version='1.0' encoding='utf-8'?>
-<tool id="megahit_contig2fastg" name="MEGAHIT Contig2fastg" version="@VERSION@">
-    <description>for converting MEGAHIT's contigs to fastg</description>
+<?xml version='0.9' encoding='utf-8'?>
```
Comment (bgruening) [inline tools/megahit_contig2fastg/megahit_contig2fastg.xml]: this XML should not be changed, or can be completly removed.

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml (hunk 1 above)
Comment (jraysajulga) [reply tools/megahit_contig2fastg/megahit_contig2fastg.xml]: Ah. That makes sense. Thanks. I removed those.

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml (hunk 2 above)
Comment (jraysajulga) [reply tools/megahit_contig2fastg/megahit_contig2fastg.xml]: Okay. Removed this as well.

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml (hunk 3 above)
Comment (jraysajulga) [reply tools/megahit_contig2fastg/megahit_contig2fastg.xml]: I like that.

File: tools/megahit_contig2fastg/megahit_contig2fastg.xml (hunk 4 above)
Comment (jraysajulga) [reply tools/megahit_contig2fastg/megahit_contig2fastg.xml]: I reverted it back to the original version.

**Comment by jraysajulga** [comment]
Once this looks satisfactory, can you add this tool to your https://usegalaxy.eu server?

**Comment by bgruening** [comment]
@jraysajulga this now goes to tools-iuc? So can we close this one?

**Comment by jraysajulga** [comment]
@bgruening Now that it's been approved, this should go on tools-iuc. Shall we remove this tool from the tools-galaxyp repo?

**Pull request closed without merging**